Sources/xcprojectlint-package/ErrorReporter.swift

```diff
@@ -36,3 +35,16 @@ public struct ErrorReporter {
     }
   }
 }
+
+extension ErrorReporter {
+  public func toStatusCode(from checkReport: Report) -> Int32 {
```

Comment:
I specifically didn't want put this transformation into `Report`, because I want to keep it decoupled from execution and reporting environment. 
`ErrorReporter` on the other hand is entitled to make this transformation.